roles/oem/tasks/vm_only_pre.yml

```
@@ -1,6 +1,18 @@
 ---
 
 # Tasks for the oem role to run at the beginning of playbook execution
+- name: Gather the package facts
+  ansible.builtin.package_facts:
+    manager: auto
+- name: Remove unneeded backgrounds
+  ansible.builtin.apt:
+    name: "{{ ansible_facts.packages | select('search', 'mint-background') | list }}"
+    state: absent
+    purge: yes
+  when:
+    - 'ansible_distribution_release not in item'
+    - 'ansible_distribution == "Linux Mint"'
```

```suggestion
- name: Remove unneeded backgrounds
  ansible.builtin.apt:
    name: "{{ ansible_facts.packages | select('search', 'mint-background') | reject('search', ansible_distribution_release) | list }}"
    state: absent
    purge: yes
  when:
    - 'ansible_distribution == "Linux Mint"'
```

Not 100% sure on this (I haven't tested it) but maybe?

---

That's about where Google was leading me. Testing that now, but it also gives me a slightly different idea to try as well

---

Thoughts on the modified version now? One call to apt, and even though it's not guarded by `distro == mint` we never did that before. If you agree, I'll squash git before we merge

---

LGTM!

---

Done